<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1322</width>
    <height>791</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="notesStackedWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>1241</width>
      <height>651</height>
     </rect>
    </property>
    <widget class="QWidget" name="pageRepresent">
     <widget class="QListWidget" name="listWidgetNotes">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>931</width>
        <height>581</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageAddNote">
     <widget class="QPushButton" name="btnAddNote">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>20</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditTitle">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>20</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelTitle1">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>30</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Title</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="labelNote">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>330</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Note</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEditNote">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>90</y>
        <width>1011</width>
        <height>541</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnRepresent">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Represent</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddNoteMenu">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>30</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
